src/UserSpaceInstrumentation/Trampoline.cpp

```diff
@@ -158,4 +160,38 @@ ErrorMessageOr<uint64_t> AllocateMemoryForTrampolines(pid_t pid, const AddressRa
   return AllocateInTracee(pid, address_range.start, size);
 }
 
+ErrorMessageOr<absl::flat_hash_set<uint64_t>> AllInstructionPointersFromProcess(pid_t pid) {
+  absl::flat_hash_set<uint64_t> result;
```

Comment:
All threads could be at exactly the same instruction, in which case we'd return a single IP, is that what we want?